tests/test_diffraction_objects.py

```diff
@@ -410,3 +410,15 @@ def test_all_array_setter():
         "Please use 'insert_scattering_quantity' to modify `all_arrays`.",
     ):
         actual_do.all_arrays = np.empty((4, 4))
+
+
+def test_copy_object():
+    do = DiffractionObject(
+        name="test",
+        wavelength=4.0 * np.pi,
+        xarray=np.array([0.0, 90.0, 180.0]),
+        yarray=np.array([1.0, 2.0, 3.0]),
+        xtype="tth",
+    )
+    copy_of_DO = do.copy()
+    assert do == copy_of_DO
```

Comment:
Utizling the object's `__eq__` nice implementation

Reply:
To complete this test also assert id(actual) != Id(initial)

Reply:
Then the copy is not pointing to the same object

Reply:
yes done below.